package.json

```diff
@@ -127,5 +127,8 @@
   },
   "browserslist": [
     "extends @instructure/browserslist-config-instui"
-  ]
+  ],
+  "dependencies": {
+    "codesandbox": "^2.2.3"
+  }
```

Comment:
I don't think you have to add the `codesandbox` dependency at the root level, adding it only to the `__docs__` package should be enough.